frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/vm-wizard-storage-modal-enhanced.tsx

```diff
@@ -84,7 +85,7 @@ const VMWizardStorageModal: React.FC<VMWizardStorageModalProps> = (props) => {
       <DiskModal
         {...restProps}
         storageClassConfigMap={storageClassConfigMap}
-        vmName={VM_TEMPLATE_NAME_PARAMETER}
+        vmName={vmName}
```

Comment:
let's do it here as well and use `'vm'` constant. We could create a constant called for example `DUMMY_VM_NAME`

It is probably better not to use the `VM_TEMPLATE_NAME_PARAMETER`  to mae it less confusing.

Reply:
Done!